src/unity/python/turicreate/visualization/show.py

```diff
@@ -96,46 +96,302 @@ def show(x, y, xlabel="X", ylabel="Y", title=None):
     Plot(plt_ref).show()
 
 def scatter(x, y, xlabel="X", ylabel="Y", title=None):
+    """
+    Plots the data in `x` on the X axis and the data in `y` on the Y axis
+    in a 2d scatterplot, and returns the resulting Plot object.
+    
+    The function supports SArrays of dtypes: int, float.
+
+    Parameters
+    ----------
+    x : SArray
+      The data to plot on the X axis of the scatter plot. 
+      Must be numeric (int/float)
+    y : SArray
+      The data to plot on the Y axis of the scatter plot. Must be the same
+      length as `x`.
+      Must be numeric (int/float)
+    xlabel : str (optional)
+      The text label for the X axis. Defaults to "X".
+    ylabel : str (optional)
+      The text label for the Y axis. Defaults to "Y".
+    title : str (optional)
+      The title of the plot. Defaults to None. If the value is None, the title
+      will be "<xlabel> vs. <ylabel>". Otherwise, the string passed in as the
+      title will be used as the plot title.
+
+    Return Value
+    ------------
+    An object of Plot that is the scatter plot
```

Comment:
This could be worded more clearly. See [activity classifier](https://github.com/apple/turicreate/blob/master/src/unity/python/turicreate/toolkits/activity_classifier/_activity_classifier.py#L90) for an example of a return value documented (with name, type, and description). Please follow that format/syntax - I'm not sure "Return Value" as written here will work in our HTML template.